axis.go

```diff
@@ -355,12 +350,22 @@ type DefaultTicks struct{}
 
 var _ Ticker = DefaultTicks{}
 
+var labels []float64
+
+var new_labels []float64
+
+var displayPrecision int
+
 // Ticks returns Ticks in a specified range
 func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	const SuggestedTicks = 3
-	if max < min {
+	labels = nil
+	ticks = nil
+
+	if max <= min {
```

Comment:
I am not sure I understand why you change this test from `max < min` to `max <= min` ?
(it's perhaps correct, but I just don't understand why)

Reply:
As you might have noticed, I have removed lines
if math.Nextafter(val, val+majorDelta) == val {
-			break
-		}
Same for the minorDelta. The only one case when it can have happened is when majorDelta = 0, right? And this is true when min = max. So, I moved that condition to min<=max which will cause panic, not break, Would you agree&